src/Serilog.Sinks.Email/Sinks/Email/Email.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Serilog.Sinks.Email.Sinks.Email
```

Comment:
Namespace should be _Serilog.Sinks.Email_ --- looks like we are missing `<RootNamespace>Serilog</RootNamespace>` in the CSPROJ, which communicates the correct default to IDEs.